recipes/dye-score/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "dye-score" %}
+{% set version = "0.1.0" %}
+{% set hash_type = "sha256" %}
+{% set hash = "0c50f781b5f65845840dfe86b69c601f81ae723ea63aed57d32e8ec4994b62e88" %}
```

The build is failing because the hash of the downloaded file is:
```suggestion
{% set hash = "7b7bee4ebda89f71bcc43def6736cec9948c2847a341a3fd4a10b9be2f27f9a3" %}
```
I assume this is a copy-and-paste mistake?

---

Yes. Indeed. Thanks for digging it out.